src/playground/app.ts

```diff
@@ -0,0 +1,53 @@
+import express from 'express';
+import path from 'path';
+import expressHandlebars from 'express-handlebars';
+
+import dotenv from 'dotenv';
+import fs from 'fs';
+
+if (fs.existsSync('.env')) {
+  console.log('Using .env file to supply config environment variables');
+  dotenv.config({ path: '.env' });
+} else {
+  console.log('Using .env.example file to supply config environment variables');
+  dotenv.config({ path: '.env.example' }); // you can delete this after you create your own .env file!
+}
+
+const app = express();
+
+app.use(
+  express.static(path.join(__dirname, 'public'), { maxAge: 31557600000 })
+);
+
+// handlebars
+app.engine(
+  'handlebars',
+  expressHandlebars({
+    layoutsDir: __dirname + '/views',
+    defaultLayout: 'main',
+  })
+);
+app.set('views', __dirname + '/views');
+app.set('view engine', 'handlebars');
```

Comment:
Do we need this? Seems like this information has been already provided to `app.engine`

Reply:
Seems so: https://www.npmjs.com/package/express-handlebars has both app.engine and app.set('view engine')